source/stylesheets/globals/imports/finescss/_fine_mixins.scss

```diff
@@ -18,6 +18,105 @@
   list-style: none;
 }
 
+// eager.io/blog/smarter-link-underlines/
+// Rewritten in SCSS from Stylus
+// Create iOS style text underlines
+// Comes in one mixin containing two additional mixins
+// each nested mixin depends on function/keyframe helper
+// @include underline-link(white, black);
+// @include underline-link(white, black, false);
+
+@mixin underline-link-shadow($color) {
+  text-shadow:
+    skew-it(1) 0 $color,
+    skew-it(-1) 0 $color,
+    0 skew-it(1) $color,
+    0 skew-it(-1) $color,
+    skew-it(2) 0 $color,
+    skew-it(-2) 0 $color,
+    skew-it(3) 0 $color,
+    skew-it(-3) 0 $color,
+    skew-it(4) 0 $color,
+    skew-it(-4) 0 $color,
+    skew-it(5) 0 $color,
+    skew-it(-5) 0 $color;
+}
+  @function skew-it($iteration) {
+    @return ($iteration * 0.03) * 1em;
+  }
+
+@mixin underline-link($background-color, $color, $transition: true) {
+  @include underline-link-shadow($background-color);
+  background-image: linear-gradient($background-color, $background-color), linear-gradient($background-color, $background-color), linear-gradient($color, $color);
+  background-size: 0.05em 1px, 0.05em 1px, 1px 1px;
+  background-repeat: no-repeat, no-repeat, repeat-x;
+  background-position: 0% 90%, 100% 90%, 0% 90%;
+  color: $color;
+  text-decoration: none;
+
+  @if $transition == true {
+    @include underline-transition($background-color);
+  }
+
+  &::selection {
```

Comment:
I was distracted, thought you were referring to the changes you were making
to JS

On Tue, May 26, 2015 at 2:27 PM, Tim Shedor <notifications@github.com>
wrote:

> In source/stylesheets/globals/imports/finescss/_fine_mixins.scss
> <https://github.com/wearefine/frob-core/pull/4#discussion_r31081685>:
>
> > +  }
> > +
> > +@mixin underline-link($background-color, $color, $transition: true) {
> > +  @include underline-link-shadow($background-color);
> > +  background-image: linear-gradient($background-color, $background-color), linear-gradient($background-color, $background-color), linear-gradient($color, $color);
> > +  background-size: 0.05em 1px, 0.05em 1px, 1px 1px;
> > +  background-repeat: no-repeat, no-repeat, repeat-x;
> > +  background-position: 0% 90%, 100% 90%, 0% 90%;
> > +  color: $color;
> > +  text-decoration: none;
> > +
> > +  @if $transition == true {
> > +    @include underline-transition($background-color);
> > +  }
> > +
> > +  &::selection {
>
> This selection block doesn't match the next selection block (color
> variables), $c-spec-white isn't declared, and the @placeholder mixin
> should be used here anyway.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wearefine/frob-core/pull/4/files#r31081685>.
>



-- 
/Travis